py/server/deephaven/table.py

```diff
@@ -694,6 +696,63 @@ def to_string(self, num_rows: int = 10, cols: Union[str, Sequence[str]] = None)
         except Exception as e:
             raise DHError(e, "table to_string failed") from e
 
+    def diff(self, table: Table, max_lines: int = 1, floating_inexact: bool = False,
```

Comment:
do we really want this as a table method and not a stand-alone function?  I know @rcaudy hates adding things to `Table`.

Reply:
I had a hard time placing this util function. As a design rule, we want to avoid having something generically named (utils, tools) that is likely to become a dumping ground for unrelated functionalities. Given that, the table module (table.py) is a natural choice for this function but we didn't have any module level functions, thus the decision to make it a method. I have made it a module level (standalone) function and renamed it to `table_diff`. Maybe just `diff` is better. But I like to be more explicit with this name because it behaves somewhat not the way other `diff` tools work.